website/pages/Page.tsx

```diff
@@ -43,6 +45,7 @@ export interface Props {
   /** @hide true */
   seo?: Section<SEOSection>;
   sections: Sections;
+  noIndexing: boolean;
```

Comment:
Can this prop be configure on the page?

I think there's a "noIndexing" in the app